tools/accuracy_checker/accuracy_checker/launcher/__init__.py

```diff
@@ -62,7 +62,13 @@
 
 from .opencv_launcher import OpenCVLauncher
 
-from .gapi_launcher import GAPILauncher
+try:
+    from .gapi_launcher import GAPILauncher
+except ImportError as import_error:
+    GAPILauncher = unsupported_launcher(
+        'g-apa',
```

Comment:
may be, it's misprint
```suggestion
        'g-api',
```